backend/helpers/helpers.go

```diff
@@ -46,8 +46,16 @@ func DBErrorHandling(error error, w http.ResponseWriter) {
 	}
 }
 
-func JsonWriter(w http.ResponseWriter, table interface{}) {
-	errJson := json.NewEncoder(w).Encode(&table)
+func JsonWriter(w http.ResponseWriter, response interface{}) {
+	switch response.(type) {
+	case string:
+		response = struct {
+			Message string `json:"message"`
+		}{
+			Message: response.(string),
+		}
+	}
+	errJson := json.NewEncoder(w).Encode(&response)
```

Comment:
@ABarroso647 can you indent the case here?

So this only executes the response = struct { ... } logic when response is a string right? Otherwise, the function is unchanged.

Reply:
![image](https://github.com/lynx-locks/lynx-web/assets/57304476/4e76e0eb-089c-417c-9fa0-359788b24eee)
copied formatting directly from golang website
but yes, the struct logic does not happen when the response is not a string